proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -26,3 +30,26 @@ message MsgRegisterAccount {
 message MsgRegisterAccountResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
 }
+
+// MsgSubmitTx defines the payload for MsgSubmitTx
+message MsgSubmitTx {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string owner         = 1;
+  string connection_id = 2 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  // Timeout height relative to the current block height.
```

Comment:
I'm not sure we should be taking in a relative height here (especially since the timestamp is labeled as absolute). I think I'd prefer external UX to just figure out the correct timeout height

Is there a specific use case we can think of which needs a relative timeout?

Reply:
i just grabbed the timeout parameters from the transfer app -- i guess the relative height might make sense if you want a time limit on the action relative to when you've submitted the tx (ie: you want to trigger a swap only within the next 24 hours)